manifests/j/JRiver/MediaCenter/29.0.19/JRiver.MediaCenter.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: JRiver.MediaCenter
+PackageVersion: 29.0.19
```

```suggestion
PackageVersion: 29
```
This must match Control Panel.